### ISSUE TITLE
REQ: Issue /save-all after X chunks, tweaks

### ISSUE THREAD

**Issue by idgarad** [gecgooden/chunkgen#55]
Might be a good idea to code in an option for /save-all command every X chunks if server operators have /save-off on by default or for backup purposes.

savex=100 (issue a /save-all every 100 chunks)

Additionally an EmptyOnly option to run only when player count = 0 might be nice.

From past experiences with chunk generators it may be a good idea to have an option for less then 1 chunk per tick, perhaps an option for every N ticks. Say a TickSkip option.

/chunkgen radius 0 0 1000 ~ true saveex=100 tickSkip=1 (Effectively every other tick)
/chunkgen radius 0 0 1000 ~ true saveex=300 tickSkip=2 EmptyOnly=true (effectively every 300 chunks generated do a force /save-all, generating a chunk every 3rd tick and only when there are no players online)

**Comment by jayyyin** [comment]
omg the tick skipping and empyOnly option would be super useful.

**Comment by gecgooden** [comment]
The empty only feature already exists in the form of the `pauseForPlayers` config field.

I'm not sure how to manage integrating with a backup mod, as different backup mods would have different commands. We're already doing the same as a save-all after every generated chunk.

**Comment by idgarad** [comment]
Exactly, instead of saving after every chunk, save after every X chunks. The tickSkip I think would be essential for throttling while the server is running.
